gallery/src/styles/NxList/NxListClickableExample.tsx

```diff
@@ -10,31 +10,41 @@ import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 
 const NxListClickableExample = () =>
   <ul className="nx-list nx-list--clickable">
-    <li className="nx-list__item" tabIndex={0}>
-      <span className="nx-list__text">Action 1</span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link">
+      <button className="nx-list__btn">
+        <span className="nx-list__text">Action 1</span>
+        <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+      </button>
     </li>
-    <li className="nx-list__item" tabIndex={0}>
-      <span className="nx-list__text nx-truncate-ellipsis">
-        This list item should be truncated at the right end edge. youtube weathered network network systemic
-        systema claymore mine voodoo god garage monofilament realism order-flow corporation car footage vinyl.
-      </span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link">
+      <button className="nx-list__btn">
+        <span className="nx-list__text nx-truncate-ellipsis">
+          This list item should be truncated at the right end edge. youtube weathered network network systemic
+          systema claymore mine voodoo
+        </span>
+        <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+      </button>
     </li>
-    <li className="nx-list__item selected" tabIndex={0}>
-      <span className="nx-list__text">This list item is selected</span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link selected" aria-selected="true">
+      <button className="nx-list__btn">
+        <span className="nx-list__text">Action 3</span>
+        <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+      </button>
     </li>
-    <li className="nx-list__item" tabIndex={0}>
-      <span className="nx-list__text">Action 4</span>
-      <span className="nx-list__subtext">
-        This is a fourth line. It includes text that might relate to the top line or might not.
-      </span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link">
+      <button className="nx-list__btn selected" aria-pressed="true">
```

Comment:
I think we need to look a little bit more into the difference between "pressed" and "selected", which element should have whichever one we want to use (I assume we won't need both) and which item in this example actually demonstrates that state. ChromeVox seems to support both the way this item uses `aria-pressed` and the way the one above uses `aria-selected` - they both get read as pressed or selected respectively.

Reply:
> I think we need to look a little bit more into the difference between "pressed" and "selected", which element should have whichever one we want to use (I assume we won't need both) and which item in this example actually demonstrates that state. ChromeVox seems to support both the way this item uses aria-pressed and the way the one above uses aria-selected - they both get read as pressed or selected respectively.

Also see the older discussions in the PR about these and `aria-current`.

Reply:
> I think we need to look a little bit more into the difference between "pressed" and "selected", which element should have whichever one we want to use (I assume we won't need both) and which item in this example actually demonstrates that state. ChromeVox seems to support both the way this item uses `aria-pressed` and the way the one above uses `aria-selected` - they both get read as pressed or selected respectively.

I picked "pressed" specifically because I felt that it matched best with the use of `<button>`'s. However there is a use case where "selected" may make more sense and that is when a list is used as a way to show chunks of content on a page (I think APL does this). In that case I think that "selected" is more appropriate.

Reply:
> I picked "pressed" specifically because I felt that it matched best with the use of <button>'s. However there is a use case where "selected" may make more sense and that is when a list is used as a way to show chunks of content on a page (I think APL does this). In that case I think that "selected" is more appropriate.

I'm not sure I understand the distinction you're making, but surely it doesn't make sense to use them both in the same example as we have here right?  And if they do each make sense in different situations, we should document when to use each.  And consider carefully which element it should be used on (the `<li>` vs the `<button`>, and also what about the lists of links?)

Reply:
> > I picked "pressed" specifically because I felt that it matched best with the use of 's. However there is a use case where "selected" may make more sense and that is when a list is used as a way to show chunks of content on a page (I think APL does this). In that case I think that "selected" is more appropriate.
> 
> I'm not sure I understand the distinction you're making, but surely it doesn't make sense to use them both in the same example as we have here right? And if they do each make sense in different situations, we should document when to use each. And consider carefully which element it should be used on (the `<li>` vs the `<button>`, and also what about the lists of links?)

Changed over to `aria-selected` here: https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/62c077029f3259c2c04e2f525505a2e7e5d5f006

Reply:
> Changed over to aria-selected here: 62c0770

Needed on the links example too

Reply:
> > Changed over to aria-selected here: [62c0770](https://github.com/sonatype/sonatype-react-shared-components/commit/62c077029f3259c2c04e2f525505a2e7e5d5f006)
> 
> Needed on the links example too

I purposefully did not put it on the link example because I don't think that the `<a>` should be used when we have a "selected" state (it should not be sticky in that way), the `<a>` should only be used for navigation.

Reply:
Per discussion in Slack moved to `aria-current` rather than `aria-selected`: https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/436eac785f56aa512bf85bfdf6668af0913ba9b9

Reply:
> I purposefully did not put it on the link example because I don't think that the <a> should be used when we have a "selected" state (it should not be sticky in that way), the <a> should only be used for navigation.

I don't follow.  It is very common for one of the links in the list to be to the current page.  Think about the gallery side nav, or the IQ Violation Details sidebar.  In that case the current link is styled in a highlighted way and IMO it should be called out differently too